commands.js

```diff
@@ -130,6 +130,7 @@ var commands = exports.commands = {
 			switch (targetCmd) {
 			case 'me':
 			case 'announce':
+				break;
 			case 'invite':
 				var targetRoomid = toId(target.substr(8));
```

Comment:
`Rooms.search(targetRoomid) || Simulator.battles[targetRoomid]`

wtf you don't need to do anything before or after `Rooms.search`. It's like no one actually knows how to use `Rooms.search` D: